db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/UnaryOperatorNode.java

```diff
@@ -915,4 +915,10 @@ public boolean isConstantOrParameterTreeNode() {
         // for count(*), operand is null
         return operand != null && operand.isConstantOrParameterTreeNode();
     }
+
+    public double getBaseOperationCost() throws StandardException { return getOperandCost(); }
+
+    protected double getOperandCost() throws StandardException {
+        return operand == null ? 0.0f : operand.getBaseOperationCost();
```

Comment:
(nitpic) 0.0f is float, you mean 0.0 (double)

Reply:
Done.